### ISSUE TITLE
ImportError: cannot import name 'lemmatize' from 'gensim.utils'

### ISSUE THREAD

**Issue by Polindem** [RaRe-Technologies/gensim#3239]
Hi! 

I'm having this issue:  **ImportError: cannot import name 'lemmatize' from 'gensim.utils'**
Could it be that you changed the name and it's not possible to import it as 'lemmatize'? Needed to post it here since I didn't find any information in the internet. Some people mentioned incompatibility with numpy though.
Thank you in advance for your response!

**What I use**:
macOS
Python 3.8.2 
[Clang 6.0 (clang-600.0.57)]
Bits 64
NumPy 1.21.1
SciPy 1.7.1
gensim 4.1.0
FAST_VERSION 1

**Comment by piskvorky** [comment]
That's right. Gensim used the Pattern library for lemmatization, but Patern proved an unstable dependency. So we dropped Pattern (and hence lemmatization) from Gensim in https://github.com/RaRe-Technologies/gensim/pull/3012. There's no `gensim.utils.lemmatize()` since Gensim 4.0.0 – gone forever.
